procedures/MT/rotator/rotator_analysis.ipynb

```diff
@@ -0,0 +1,918 @@
+{
```

Comment:
<p>Thanks this all looks much better. I didn't realize this on my first pass but you are using bold font in your titles instead of actual titles.</p><p>If you use actual titles then there is a summary rendered with click-able links that observers have requested. </p><p>I'll send you a screenshot via Slack so you know what I mean... it's hard to explain here.</p><p> </p><p>So for a title use a single #, for a subsection heading use ##</p><p> </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-ts/ts_notebooks/pull/96/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
Fixed the commit: [30d19d5](https://github.com/lsst-ts/ts_notebooks/pull/96/commits/30d19d514c64981689c466b89131d18ff78a1464). Thanks!